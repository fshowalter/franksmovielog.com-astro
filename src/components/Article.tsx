import type { StillImageData } from "src/api/stills";

import { LongFormText } from "./LongFormText";
import { PageTitle } from "./PageTitle";
import { StillList } from "./StillList";
import { StillListHeading } from "./StillListHeading";
import type { StillListItemData } from "./StillListItem";
import { StillListNav } from "./StillListNav";

export const StillImageConfig = {
  width: 960,
  height: 540,
  sizes: "(min-width: 960px) 960px, 100vw",
};

export function Article({
  alt,
  title,
  content,
  moreReviewsData,
  moreReviewsStills,
  imageData,
}: {
  alt: string;
  content: string | null;
  title: string;
  imageData: StillImageData;
  moreReviewsData: StillListItemData[];
  moreReviewsStills: Record<string, StillImageData>;
}): JSX.Element {
  return (
    <main>
      <article className="flex flex-col items-center">
        <PageTitle className="px-pageMargin py-6 text-center desktop:py-8">
          {title}
        </PageTitle>
        <img
          {...imageData}
          width={StillImageConfig.width}
          height={StillImageConfig.height}
          sizes={StillImageConfig.sizes}
          loading="lazy"
          decoding="async"
          alt={alt}
          className="mb-[5.33px]"
        />
        <div className="spacer-y-16" />
        <div className="px-pageMargin">
          <LongFormText text={content} className="max-w-prose" />
        </div>
        <div className="spacer-y-32" />
      </article>
      <div className="flex w-full max-w-popout items-center justify-center bg-default tablet:max-w-full tablet:bg-subtle tablet:pb-32 tablet:pt-8">
        <StillListNav>
          <StillListHeading
            leadText="Latest"
            linkText="Reviews"
            linkTarget={`/reviews/`}
          />
          <StillList
            stills={moreReviewsStills}
            titles={moreReviewsData}
            seeAllLinkTarget="/reviews/"
            seeAllLinkText="Reviews"
          />
        </StillListNav>
      </div>
    </main>
  );
}
