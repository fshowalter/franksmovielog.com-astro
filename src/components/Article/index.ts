export { Article } from "./Article";
export { getProps } from "./getProps";
