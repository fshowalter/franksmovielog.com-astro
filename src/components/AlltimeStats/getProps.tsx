import { alltimeStats } from "src/api/alltimeStats";
import { getFluidWidthPosters, getFixedWidthPosters } from "src/api/posters";
import { MostWatchedMoviesPosterConfig } from "src/components/MostWatchedMovies";
import { ListItemPosterImageConfig } from "src/components/ListItemPoster";
import { allStatYears } from "src/api/yearStats";
import { type Props } from "./AlltimeStats";

export async function getProps(): Promise<Props> {
  const stats = await alltimeStats();
  const distinctStatYears = await allStatYears();

  const mostWatchedPeoplePosters = await getFixedWidthPosters(
    ListItemPosterImageConfig,
  );
  const mostWatchedMoviesPosters = await getFluidWidthPosters(
    MostWatchedMoviesPosterConfig,
  );

  return {
    stats,
    distinctStatYears,
    mostWatchedMoviesPosters,
    mostWatchedPeoplePosters,
  };
}
