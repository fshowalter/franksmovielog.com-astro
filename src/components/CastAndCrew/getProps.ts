import { type ListItemValue } from "./List";
import { allCastAndCrew } from "src/api/castAndCrew";
import { getAvatars } from "src/api/avatars";
import { ListItemAvatarImageConfig } from "src/components/ListItemAvatar";
import type { Props } from "./CastAndCrew";

export async function getProps(): Promise<Props> {
  const { castAndCrew } = await allCastAndCrew();
  const avatars = await getAvatars(ListItemAvatarImageConfig);

  castAndCrew.sort((a, b) => a.name.localeCompare(b.name));

  const values = castAndCrew.map((member) => {
    const value: ListItemValue = {
      name: member.name,
      slug: member.slug,
      reviewCount: member.reviewCount,
      totalCount: member.totalCount,
      creditedAs: member.creditedAs,
    };

    return value;
  });

  return { values, avatars, initialSort: "name-asc" };
}
