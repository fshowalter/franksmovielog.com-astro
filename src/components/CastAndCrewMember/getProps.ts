import { getAvatars } from "src/api/avatars";
import { allCastAndCrew } from "src/api/castAndCrew";
import { getFixedWidthPosters } from "src/api/posters";
import { ListItemPosterImageConfig } from "src/components/ListItemPoster";

import type { Props } from "./CastAndCrewMember";
import { AvatarImageConfig } from "./Header";

export async function getProps(slug: string): Promise<Props> {
  const { castAndCrew, distinctReleaseYears } = await allCastAndCrew();

  const member = castAndCrew.find((member) => {
    return slug == member.slug;
  })!;

  member.titles.sort((a, b) =>
    a.releaseSequence.localeCompare(b.releaseSequence),
  );

  const posters = await getFixedWidthPosters(ListItemPosterImageConfig);
  const avatars = await getAvatars(AvatarImageConfig);
  const avatarImageData = avatars[member.slug];

  return {
    value: member,
    posters,
    avatarImageData,
    distinctReleaseYears,
    initialSort: "release-date-asc",
  };
}
