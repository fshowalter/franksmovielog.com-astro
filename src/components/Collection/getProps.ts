import { getAvatars } from "src/api/avatars";
import { allCollections } from "src/api/collections";
import { getFixedWidthPosters } from "src/api/posters";
import { ListItemPosterImageConfig } from "src/components/ListItemPoster";

import type { Props } from "./Collection";
import { AvatarImageConfig } from "./Header";

export async function getProps(slug: string): Promise<Props> {
  const { collections, distinctReleaseYears } = await allCollections();

  const member = collections.find((member) => {
    return slug == member.slug;
  })!;

  member.titles.sort((a, b) =>
    a.releaseSequence.localeCompare(b.releaseSequence),
  );

  const posters = await getFixedWidthPosters(ListItemPosterImageConfig);
  const avatars = await getAvatars(AvatarImageConfig);
  const avatarImageData = avatars[member.slug];

  return {
    value: member,
    posters,
    avatarImageData,
    distinctReleaseYears,
    initialSort: "release-date-asc",
  };
}
