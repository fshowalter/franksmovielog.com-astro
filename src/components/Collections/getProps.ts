import { getAvatars } from "src/api/avatars";
import { allCollections } from "src/api/collections";
import { ListItemAvatarImageConfig } from "src/components/ListItemAvatar";

import type { Props } from "./Collections";
import { type ListItemValue } from "./List";

export async function getProps(): Promise<Props> {
  const { collections } = await allCollections();
  const avatars = await getAvatars(ListItemAvatarImageConfig);

  collections.sort((a, b) => a.name.localeCompare(b.name));

  const listItemValues = collections.map((member) => {
    const listItemValue: ListItemValue = {
      name: member.name,
      slug: member.slug,
      reviewCount: member.reviewCount,
      titleCount: member.titleCount,
    };

    return listItemValue;
  });

  return { values: listItemValues, avatars, initialSort: "name-asc" };
}
