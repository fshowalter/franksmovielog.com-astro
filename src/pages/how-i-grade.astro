---
import { Article } from "src/components/Article/Article";
import { getProps } from "src/components/Article/getProps";
import Layout from "src/layouts/Layout.astro";

const props = await getProps({
  slug: "how-i-grade",
  alt: "Empty cinema seats.",
});
---

<Layout
  title={props.title}
  description="The criteria I use to rate movies on this site."
  article
>
  <Article {...props} />
</Layout>
