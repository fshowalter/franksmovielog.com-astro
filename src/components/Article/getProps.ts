import { getPage } from "src/api/pages";
import { allReviews } from "src/api/reviews";
import { getStills } from "src/api/stills";
import { StillListItemImageConfig } from "src/components/StillListItem";

import type { Props } from "./Article";
import { StillImageConfig } from "./Article";

export async function getProps({
  slug,
  alt,
}: {
  slug: string;
  alt: string;
}): Promise<Props> {
  const { title, content } = await getPage(slug);

  const { reviews } = await allReviews();

  reviews.sort((a, b) => b.sequence.localeCompare(a.sequence));

  const moreReviewsValues = reviews.slice(0, 4);

  const stills = await getStills(StillImageConfig);
  const imageData = stills[slug];

  const moreReviewsStills = await getStills(StillListItemImageConfig);

  return {
    title,
    content,
    alt,
    moreReviewsStills,
    imageData,
    moreReviewsValues,
  };
}
