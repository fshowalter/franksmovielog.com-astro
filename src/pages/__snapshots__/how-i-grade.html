<!DOCTYPE html><html lang="en"> <head><meta charset="UTF-8"><meta name="viewport" content="width=device-width"><link rel="icon" href="/favicon.ico" sizes="48x48"><link rel="icon" href="/favicon.svg" sizes="any" type="image/svg+xml"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="manifest" href="/site.webmanifest"><meta name="description" content="The criteria I use to rate movies on this site."><meta name="og:image"><meta property="og:url" content="https://www.franksmovielog.com//"><meta property="og:type" content="article"><meta property="og:title" content="How I Grade"><meta property="og:description" content="The criteria I use to rate movies on this site."><title>How I Grade | Frank&#39;s Movie Log</title></head>  <div><a href="#content" class="translate-skip-to-content absolute left-1/2 top-0.5 z-50 mx-auto bg-subtle px-6 py-2 text-center text-accent">Skip to content</a><div class="image-filter mx-auto min-h-4 w-full max-w-canvas bg-notcomingsoon desktop:sticky desktop:top-0 desktop:z-40"></div><div class="mx-auto flex min-h-full max-w-canvas flex-col bg-default"><header class="border-bottom flex flex-col items-center gap-6 bg-default px-pageMargin py-6 text-center desktop:sticky desktop:top-4 desktop:z-40 desktop:flex-row desktop:flex-wrap desktop:justify-between desktop:py-8 desktop:text-left"><div class="items-inherit justify-items-inherit flex flex-col"><h1 class="whitespace-nowrap font-normal leading-8" style="font-size:1.5625rem"><a class="text-default" href="/">Frank&#x27;s Movie Log</a></h1><p class="w-full text-sm italic leading-4 text-muted desktop:pl-px">My life at the movies.</p></div><div class="w-full max-w-prose desktop:w-auto max:order-4"><form action="https://www.google.com/search" accept-charset="UTF-8" method="get" role="search" class="border-all w-full max-w-prose overflow-hidden rounded"><label for="search" class="flex overflow-hidden"><span class="sr-only">Search</span><input type="text" name="q" id="search" placeholder="Search..." class="min-w-0 grow border-0 bg-subtle px-4 py-2 text-base font-light leading-6 text-default"/><input type="hidden" name="q" value="site:www.franksmovielog.com"/><button type="submit" value="Search" aria-label="Search" class="border-0 border-l border-default bg-subtle px-3 py-1"><svg class="size-4 fill-subtle" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M10.442 10.442a1 1 0 011.415 0l3.85 3.85a1 1 0 01-1.414 1.415l-3.85-3.85a1 1 0 010-1.415z"></path><path fill-rule="evenodd" d="M6.5 12a5.5 5.5 0 100-11 5.5 5.5 0 000 11zM13 6.5a6.5 6.5 0 11-13 0 6.5 6.5 0 0113 0z"></path></svg></button></label></form></div><nav class="w-full max:w-auto"><ul class="flex flex-wrap justify-center gap-x-4 gap-y-2 text-accent tablet:gap-x-6 desktop:justify-start"><li class="block whitespace-nowrap tracking-0.5px"><a class="text-muted" href="/">Home</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/how-i-grade/">How I Grade</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/reviews/">Reviews</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/viewings/">Viewing Log</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/cast-and-crew/">Cast &amp; Crew</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/collections/">Collections</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/watchlist/">Watchlist</a></li></ul></nav></header><div class="grow" id="content">  <main><article class="flex flex-col items-center"><h1 class="text-[2rem] font-normal leading-none desktop:text-[2.25rem] px-pageMargin py-6 text-center desktop:py-8">How I Grade</h1><img srcSet="/_image?/content%2Fassets%2Fstills%2Fhow-i-grade.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=240&amp;q=80&amp;f=avif 240w, /_image?/content%2Fassets%2Fstills%2Fhow-i-grade.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=480&amp;q=80&amp;f=avif 480w, /_image?/content%2Fassets%2Fstills%2Fhow-i-grade.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=960&amp;q=80&amp;f=avif 960w, /_image?/content%2Fassets%2Fstills%2Fhow-i-grade.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=1920&amp;q=80&amp;f=avif 1920w" src="/_image?/content%2Fassets%2Fstills%2Fhow-i-grade.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=960&amp;h=540&amp;q=80&amp;f=avif" width="960" height="540" sizes="(min-width: 960px) 960px, 100vw" loading="lazy" decoding="async" alt="Empty cinema seats." class="mb-[5.33px]"/><div class="spacer-y-16"></div><div class="px-pageMargin"><div class="rendered-markdown font-light text-md/7 tracking-0.3px tablet:text-xl/8 max-w-prose"><ul>
<li>
<h2>a great movie</h2>
<p><img src="/svg/5-stars.svg" alt="5 stars"> I loved it. A must-see. A perfect or almost perfect movie. These are the movies I recommend to everyone. A hard rating to earn.</p>
</li>
<li>
<h2>a good movie</h2>
<p><img src="/svg/4-stars.svg" alt="4 stars"> I really liked it. Unless you hate the genre or star, watch it. These are the movies I recommend to most folks. They may not resonate like the 5-star selections, but they don’t disappoint.</p>
</li>
<li>
<h2>an okay movie</h2>
<p><img src="/svg/3-stars.svg" alt="3 stars"> I liked it. I wouldn’t recommend it, but I wouldn’t recommend <em>against</em> it either. Most movies fall into this category.</p>
</li>
<li>
<h2>a bad movie</h2>
<p><img src="/svg/2-stars.svg" alt="2 stars"> I didn’t like it. Unless you have a deep-seated love for the genre or star, avoid it. It wasn’t awful, but if I paid money, I felt burned.</p>
</li>
<li>
<h2>an awful movie</h2>
<p><img src="/svg/1-star.svg" alt="1 star"> I hated it. Everyone involved should apologize. Another a hard rating to earn. I can forgive a movie many things. Bad acting? It happens. Specious plotting? Maybe it’s atmosphere. Gratuitous nudity? Sure. But I can’t forgive boring. When reading a 1-star review, remember: I watch them so you don’t have to.</p>
</li>
</ul>
<hr>
<h2>On Grading</h2>
<p>Let’s start with a binary scale. Gene Siskel argued a person yearns to know: “Should I watch this movie?”<sup><a href="#user-content-fn-1" id="user-content-fnref-1" data-footnote-ref="" aria-describedby="footnote-label">1</a></sup></p>
<p>A pragmatic approach, but one that doesn’t differentiate between movies one <em>should</em> see and movies one <em>must</em> see. <em>Batman Begins</em>  and <a href="/reviews/the-dark-knight-2008/"><em>The Dark Knight</em></a> both deserve a “Thumbs up” but they are not equal.</p>
<p>Enter the star scale. I’ve encountered scales from three stars to ten, sometimes with half-stars.</p>
<p>But not all star-systems prove equal.</p>
<p>Roger Ebert hated the four-star scale his paper enforced because it presented no middle ground.<sup><a href="#user-content-fn-2" id="user-content-fnref-2" data-footnote-ref="" aria-describedby="footnote-label">2</a></sup> Awful, bad, good, or great. No room for okay. Thus, we can rule out the four-star system.</p>
<p>Next, let’s consider the IMDb. It proffers a ten-star system. That’s a lot of stars. Perhaps too many. In 2009 the median movie score hit 6.6, implying users skew toward the upper bounds. Ergo, if you hate a 1-star and love a 10-star, what does a 3-star mean?</p>
<p>So if a four-star scale proves too small and ten-star scale too large, what’s ideal?</p>
<p>I like a five-star system. Ebert liked it too.<sup><a href="#user-content-fn-1" id="user-content-fnref-1-2" data-footnote-ref="" aria-describedby="footnote-label">1</a></sup> It offers a middle ground (3-stars) while proffering discreet rating definitions.</p>
<p>And what about letter grade systems?<sup><a href="#user-content-fn-3" id="user-content-fnref-3" data-footnote-ref="" aria-describedby="footnote-label">3</a></sup> I like them. Under-the-covers, they translate to five-star systems. A’s equate to 5-stars, B’s to 4-stars, and so on. I employ letter grades behind the scenes because it’s how my brain works.</p>
<p>So why display them as stars? Well, for one, grade definitions aren’t universal. To me, a ‘C’ equates to an “okay” movie, to others it screams burn. A 3-star out of 5 display indicates a middle-of-the-road rating.</p>
<p>Grading systems are hard. I think a 5-star system works, but I’m not married to the idea. Some of my favorite reviewers eschew grades, preferring you read the review. Fair enough, but I bet they rate them behind the scenes.</p>
<section data-footnotes="" class="footnotes"><h2 class="sr-only" id="footnote-label">Footnotes</h2>
<ol>
<li id="user-content-fn-1">
<p><a href="http://www.rogerebert.com/rogers-journal/you-give-out-too-many-stars">Ebert, Roger, “You Give Out Too Many Stars”, Roger Ebert , last modified September 14, 2008</a> <a href="#user-content-fnref-1" data-footnote-backref="" aria-label="Back to reference 1" class="data-footnote-backref">↩</a> <a href="#user-content-fnref-1-2" data-footnote-backref="" aria-label="Back to reference 1-2" class="data-footnote-backref">↩<sup>2</sup></a></p>
</li>
<li id="user-content-fn-2">
<p>Ebert also argues that readers dislike scales with a middle ground, but I disagree. We know what it means for a movie to be “okay.” Sometimes, instead of “Should I see this movie?” readers want to know “Will I hate this movie?” A middle ground answers that question. <a href="#user-content-fnref-2" data-footnote-backref="" aria-label="Back to reference 2" class="data-footnote-backref">↩</a></p>
</li>
<li id="user-content-fn-3">
<p><a href="http://www.avclub.com/film/">The Onion A.V. Club</a> makes great use of letter grades. <a href="#user-content-fnref-3" data-footnote-backref="" aria-label="Back to reference 3" class="data-footnote-backref">↩</a></p>
</li>
</ol>
</section></div></div><div class="spacer-y-32"></div></article><div class="flex w-full max-w-popout items-center justify-center bg-default tablet:max-w-full tablet:bg-subtle tablet:pb-32 tablet:pt-8"><nav class="relative flex w-full flex-col items-center tablet:w-auto"><div class="border-bottom w-full px-gutter py-2 tablet:py-4 tablet:shadow-none desktop:px-pageMargin desktop:py-4"><span class="font-semibold text-muted">Latest<!-- --> </span><a href="/reviews/">Reviews</a></div><ul class="w-full tablet:grid tablet:w-auto tablet:grid-cols-[repeat(2,minmax(100px,312px))] tablet:gap-8 tablet:px-gutter desktop:max-w-unset desktop:grid-cols-[repeat(4,1fr)] desktop:px-pageMargin desktop:pt-2"><li class="flow-root w-full px-gutter py-6 even:bg-subtle tablet:grid tablet:max-w-[312px] tablet:grid-rows-[auto_auto_1fr_auto] tablet:gap-y-2 tablet:overflow-hidden tablet:rounded-lg tablet:bg-default tablet:p-0 tablet:pb-8 tablet:shadow-all tablet:shadow-border tablet:even:bg-default"><div class="row-start-1 row-end-1"><a href="/reviews/chamber-of-horrors-1966/" class="safari-border-radius-fix float-right mb-2 ml-6 block w-[calc(50%_-_12px)] max-w-[312px] overflow-hidden rounded-lg tablet:float-none tablet:m-0 tablet:w-auto tablet:rounded-none"><img srcSet="/_image?/content%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=78&amp;q=80&amp;f=avif 78w, /_image?/content%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=156&amp;q=80&amp;f=avif 156w, /_image?/content%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;q=80&amp;f=avif 312w, /_image?/content%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=624&amp;q=80&amp;f=avif 624w" src="/_image?/content%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;h=176&amp;q=80&amp;f=avif" alt="A still from Chamber of Horrors (1966)" loading="lazy" decoding="async" width="312" height="175.5" sizes="(min-width: 706px) 312px, (min-width: 1280) 25vw, (min-width: 1472px) 312px, 50vw" class="aspect-[1.77777778] h-auto"/></a></div><div class="row-start-2 mb-2 tablet:m-0 tablet:px-6"><a href="/reviews/chamber-of-horrors-1966/" class="block text-md text-default">Chamber of Horrors<!-- --> <span class="text-sm font-light leading-none text-muted">1966</span></a></div><div class="mb-6 tablet:m-0 tablet:px-6 tablet:pb-6"><picture><source srcSet="/svg/2-stars-dark.svg" media="(prefers-color-scheme: dark)"/><img src="/svg/2-stars.svg" alt="D+: 2 stars (out of 5)" height="16" width="80"/></picture></div><div class="tablet:px-6"><div class="text-sm leading-4 tracking-0.5px text-subtle"><span>Crime</span><span> | <!-- -->Horror</span><span> | <!-- -->Thriller</span></div></div></li><li class="flow-root w-full px-gutter py-6 even:bg-subtle tablet:grid tablet:max-w-[312px] tablet:grid-rows-[auto_auto_1fr_auto] tablet:gap-y-2 tablet:overflow-hidden tablet:rounded-lg tablet:bg-default tablet:p-0 tablet:pb-8 tablet:shadow-all tablet:shadow-border tablet:even:bg-default"><div class="row-start-1 row-end-1"><a href="/reviews/two-for-the-seesaw-1962/" class="safari-border-radius-fix float-right mb-2 ml-6 block w-[calc(50%_-_12px)] max-w-[312px] overflow-hidden rounded-lg tablet:float-none tablet:m-0 tablet:w-auto tablet:rounded-none"><img srcSet="/_image?/content%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=78&amp;q=80&amp;f=avif 78w, /_image?/content%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=156&amp;q=80&amp;f=avif 156w, /_image?/content%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;q=80&amp;f=avif 312w, /_image?/content%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=624&amp;q=80&amp;f=avif 624w" src="/_image?/content%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;h=176&amp;q=80&amp;f=avif" alt="A still from Two for the Seesaw (1962)" loading="lazy" decoding="async" width="312" height="175.5" sizes="(min-width: 706px) 312px, (min-width: 1280) 25vw, (min-width: 1472px) 312px, 50vw" class="aspect-[1.77777778] h-auto"/></a></div><div class="row-start-2 mb-2 tablet:m-0 tablet:px-6"><a href="/reviews/two-for-the-seesaw-1962/" class="block text-md text-default">Two for the Seesaw<!-- --> <span class="text-sm font-light leading-none text-muted">1962</span></a></div><div class="mb-6 tablet:m-0 tablet:px-6 tablet:pb-6"><picture><source srcSet="/svg/3-stars-dark.svg" media="(prefers-color-scheme: dark)"/><img src="/svg/3-stars.svg" alt="C+: 3 stars (out of 5)" height="16" width="80"/></picture></div><div class="tablet:px-6"><div class="text-sm leading-4 tracking-0.5px text-subtle"><span>Drama</span><span> | <!-- -->Romance</span></div></div></li><li class="flow-root w-full px-gutter py-6 even:bg-subtle tablet:grid tablet:max-w-[312px] tablet:grid-rows-[auto_auto_1fr_auto] tablet:gap-y-2 tablet:overflow-hidden tablet:rounded-lg tablet:bg-default tablet:p-0 tablet:pb-8 tablet:shadow-all tablet:shadow-border tablet:even:bg-default"><div class="row-start-1 row-end-1"><a href="/reviews/the-man-with-nine-lives-1940/" class="safari-border-radius-fix float-right mb-2 ml-6 block w-[calc(50%_-_12px)] max-w-[312px] overflow-hidden rounded-lg tablet:float-none tablet:m-0 tablet:w-auto tablet:rounded-none"><img srcSet="/_image?/content%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=78&amp;q=80&amp;f=avif 78w, /_image?/content%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=156&amp;q=80&amp;f=avif 156w, /_image?/content%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;q=80&amp;f=avif 312w, /_image?/content%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=624&amp;q=80&amp;f=avif 624w" src="/_image?/content%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;h=176&amp;q=80&amp;f=avif" alt="A still from The Man with Nine Lives (1940)" loading="lazy" decoding="async" width="312" height="175.5" sizes="(min-width: 706px) 312px, (min-width: 1280) 25vw, (min-width: 1472px) 312px, 50vw" class="aspect-[1.77777778] h-auto"/></a></div><div class="row-start-2 mb-2 tablet:m-0 tablet:px-6"><a href="/reviews/the-man-with-nine-lives-1940/" class="block text-md text-default">The Man with Nine Lives<!-- --> <span class="text-sm font-light leading-none text-muted">1940</span></a></div><div class="mb-6 tablet:m-0 tablet:px-6 tablet:pb-6"><picture><source srcSet="/svg/2-half-stars-dark.svg" media="(prefers-color-scheme: dark)"/><img src="/svg/2-half-stars.svg" alt="C-: 2.5 stars (out of 5)" height="16" width="80"/></picture></div><div class="tablet:px-6"><div class="text-sm leading-4 tracking-0.5px text-subtle"><span>Horror</span><span> | <!-- -->Mystery</span><span> | <!-- -->Sci-Fi</span></div></div></li><li class="flow-root w-full px-gutter py-6 even:bg-subtle tablet:grid tablet:max-w-[312px] tablet:grid-rows-[auto_auto_1fr_auto] tablet:gap-y-2 tablet:overflow-hidden tablet:rounded-lg tablet:bg-default tablet:p-0 tablet:pb-8 tablet:shadow-all tablet:shadow-border tablet:even:bg-default"><div class="row-start-1 row-end-1"><a href="/reviews/burial-ground-the-nights-of-terror-1981/" class="safari-border-radius-fix float-right mb-2 ml-6 block w-[calc(50%_-_12px)] max-w-[312px] overflow-hidden rounded-lg tablet:float-none tablet:m-0 tablet:w-auto tablet:rounded-none"><img srcSet="/_image?/content%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=78&amp;q=80&amp;f=avif 78w, /_image?/content%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=156&amp;q=80&amp;f=avif 156w, /_image?/content%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;q=80&amp;f=avif 312w, /_image?/content%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=624&amp;q=80&amp;f=avif 624w" src="/_image?/content%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=312&amp;h=176&amp;q=80&amp;f=avif" alt="A still from Burial Ground: The Nights of Terror (1981)" loading="lazy" decoding="async" width="312" height="175.5" sizes="(min-width: 706px) 312px, (min-width: 1280) 25vw, (min-width: 1472px) 312px, 50vw" class="aspect-[1.77777778] h-auto"/></a></div><div class="row-start-2 mb-2 tablet:m-0 tablet:px-6"><a href="/reviews/burial-ground-the-nights-of-terror-1981/" class="block text-md text-default">Burial Ground: The Nights of Terror<!-- --> <span class="text-sm font-light leading-none text-muted">1981</span></a></div><div class="mb-6 tablet:m-0 tablet:px-6 tablet:pb-6"><picture><source srcSet="/svg/3-stars-dark.svg" media="(prefers-color-scheme: dark)"/><img src="/svg/3-stars.svg" alt="C: 3 stars (out of 5)" height="16" width="80"/></picture></div><div class="tablet:px-6"><div class="text-sm leading-4 tracking-0.5px text-subtle"><span>Horror</span></div></div></li><li class="col-[1_/_-1] block px-gutter py-4 text-right shadow-bottom shadow-border tablet:absolute tablet:right-0 tablet:top-0 tablet:shadow-none desktop:right-[var(--gutter-width)]"><a href="/reviews/">All<!-- --> <span class="inline tablet:hidden desktop:inline">Reviews</span> <!-- -->→</a></li></ul></nav></div></main>  </div><footer class="flex flex-col items-center gap-y-6 bg-notcomingsoon px-pageMargin py-8 text-inverse"><ul class="flex w-full flex-wrap justify-center gap-x-4 gap-y-2 text-inverse tablet:gap-x-6 max:w-auto"><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inverse" href="/">Home</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/how-i-grade/">How I Grade</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/reviews/">Reviews</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/viewings/">Viewing Log</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/cast-and-crew/">Cast &amp; Crew</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/collections/">Collections</a></li><li class="block whitespace-nowrap tracking-0.5px"><a class="text-inherit" href="/watchlist/">Watchlist</a></li></ul><p class="text-sm font-light leading-4">All stills used in accordance with the<!-- --> <a href="http://www.copyright.gov/title17/92chap1.html#107" class="text-inherit">Fair Use Law.</a></p><a href="#top" class="sr-only">To the top ↑</a></footer></div></div> </html>